scripts/publish-docs.js

```diff
@@ -16,10 +12,16 @@ if (process.env.TRAVIS === 'true') {
   shelljs.exec(`git config --global user.email "${userEmail}"`);
   shelljs.exec(`git config --global user.name "${userName}"`);
   shelljs.exec(`git remote add ${docsOrigin} ${remoteUrl} > /dev/null 2>&1`);
+} else {
+  shelljs.exec(
+    `git remote add ${docsOrigin} git@github.com:commercetools/ui-kit.git > /dev/null 2>&1`
```

Comment:
This part is in case you run the script locally, in which case you don't need the `GH_TOKEN` and can simply use the ssh remote.

Reply:
This seems like a useful comment in the script :)